package.json

```diff
@@ -1,47 +1,57 @@
 {
-  "name": "examp-ng-app",
-  "version": "0.0.0",
-  "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
-  },
-  "private": true,
-  "dependencies": {
-    "@angular/animations": "~8.2.13",
-    "@angular/common": "~8.2.13",
-    "@angular/compiler": "~8.2.13",
-    "@angular/core": "~8.2.13",
-    "@angular/forms": "~8.2.13",
-    "@angular/platform-browser": "~8.2.13",
-    "@angular/platform-browser-dynamic": "~8.2.13",
-    "@angular/router": "~8.2.13",
-    "rxjs": "~6.4.0",
-    "tslib": "^1.10.0",
-    "zone.js": "~0.9.1"
-  },
-  "devDependencies": {
-    "@angular-devkit/build-angular": "~0.803.17",
-    "@angular/cli": "~8.3.17",
-    "@angular/compiler-cli": "~8.2.13",
-    "@angular/language-service": "~8.2.13",
-    "@types/node": "~8.9.4",
-    "@types/jasmine": "~3.3.8",
-    "@types/jasminewd2": "~2.0.3",
-    "codelyzer": "^5.0.0",
-    "jasmine-core": "~3.4.0",
-    "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~4.1.0",
-    "karma-chrome-launcher": "~2.2.0",
-    "karma-coverage-istanbul-reporter": "~2.0.1",
-    "karma-jasmine": "~2.0.1",
-    "karma-jasmine-html-reporter": "^1.4.0",
-    "protractor": "~5.4.0",
-    "ts-node": "~7.0.0",
-    "tslint": "~5.15.0",
-    "typescript": "~3.5.3"
-  }
-}
+    "name": "examp-ng-app",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "start": "ng serve",
+        "build": "ng build",
+        "test": "ng test",
+        "e2e": "ng e2e",
+        "format:fix": "pretty-quick --staged",
+        "lint": "ng lint",
+        "format:check": "prettier --config ./.prettierrc --list-different \"src/{app,environments,assets}/**/*{.ts,.js,.json,.css,.scss}\""
+    },
+    "private": true,
+    "dependencies": {
+        "@angular/animations": "~8.2.13",
+        "@angular/cdk": "8.2.3",
+        "@angular/common": "~8.2.13",
+        "@angular/compiler": "~8.2.13",
+        "@angular/core": "~8.2.13",
+        "@angular/forms": "~8.2.13",
+        "@angular/material": "^8.2.3",
+        "@angular/platform-browser": "~8.2.13",
+        "@angular/platform-browser-dynamic": "~8.2.13",
+        "@angular/router": "~8.2.13",
+        "angular-material": "^1.1.20",
+        "ng-angular8-datatable": "^0.9.11",
```

Comment:
There are still `^` and `~` being used in packages versions. Please remove them.